spec/team_spec.rb

```diff
@@ -4,14 +4,26 @@
    before(:each) do
       @teams = []
       CSV.foreach('data/teams_fixture.csv', headers: true, header_converters: :symbol) do |row|
-         team_id = row[:id]
-         team_name = row[:teamName]
+         team_id = row[:team_id]
+         team_name = row[:team_name]
          @teams << Team.new(team_id, team_name)
       end
       @teams
+      @team = @teams[0]
    end
 
-   it 'exists' do
-      expect(@teams[0]).to be_a Team
+   describe 'Team' do
+      it 'exists' do
+         expect(@teams[0]).to be_a Team
+      end
+
+      it 'has a team id' do
```

Comment:
Yeap! I like this better. I've seen people do everything together but I like this better because it's more clear what we are testing.